.github/workflows/test.yml

```diff
@@ -19,40 +18,44 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # pytest-split automatically distributes work load so parallel jobs finish in similar time
         os: [ubuntu-latest]
         python-version: ["3.11"]
 
     runs-on: ${{ matrix.os }}
 
     env:
-      PMG_MAPI_KEY: ${{ secrets.PMG_MAPI_KEY }}
-      PMG_VASP_PSP_DIR: /home/runner/work/matgenb/matgenb/psp
+      MP_API_KEY: ${{ secrets.PMG_MAPI_KEY }}
```

Comment:
Need to double check this API key env var, setting it as `PMG_MAPI_KEY` doesn't seem to be picked up correctly

As I don't have access to repo secrets, not sure if it's a legacy key or new key